### PR TITLE
3ds2 global support

### PR DESCRIPTION
**Pull request by DavidRadchenko**

### CODE REVIEW

**Review by the-fine — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -44,6 +44,8 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
```
Comment (julia-elikhis) [inline pom.xml:48]: can we set target to 11? We will soon need this module in commerce service

File: src/main/java/cc/protea/spreedly/model/SpreedlyAmexNetwork.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package cc.protea.spreedly.model;
+
+import javax.xml.bind.annotation.*;
+
+@XmlRootElement(name = "amex")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class SpreedlyAmexNetwork {
+
+	@XmlElement(name = "acquirer_merchant_id") public String acquirerMerchantId;
+	@XmlElement(name = "country_code") public Long countryCode;
```
Comment (julia-elikhis) [inline src/main/java/cc/protea/spreedly/model/SpreedlyAmexNetwork.java:10]: can be Integer, but not critical

**Comment by julia-elikhis** [comment]
LG, let's take the java 11 compatibility question to the next pr

**Review by julia-elikhis — APPROVED** [review]

**Pull request merged**